### PR TITLE
Bump versions to v0.1.1

### PR DESCRIPTION
**Pull request by zytx121**
#### New Features

- Support [huge image inference](deom/huge_image_demo.py) (#34)
- Support HRSC Dataset (#96)
- Support mixed precision training (#72)
- Add [colab tutorial](demo/MMRotate_Tutorial.ipynb) for beginners (#66)
- Add inference speed statistics [tool](tools/analysis_tools/benchmark.py) (#86)
- Add confusion matrix analysis [tool](tools/analysis_tools/confusion_matrix.py) (#93)

#### Bug Fixes

- Fix URL error of Swin pretrained model (#111)
- Fix bug for SASM durning training (#105)
- Fix rbbox_overlaps abnormal when the box is too small (#61)
- Fix bug for visualization (#12, #81)
- Fix stuck when compute mAP (#14, #52)
- Fix 'RoIAlignRotated' object has no attribute 'out_size' bug (#51)
- Add missing init_cfg in dense head (#37)
- Fix install an additional mmcv (#17)
- Fix typos in docs (#3, #11, #36)

#### Improvements

- Move `eval_rbbox_map` from `mmrotate.datasets` to `mmrotate.core.evaluation` (#73)
- Add  Windows CI (#31)
- Add copyright commit hook (#30)
- Add Chinese translation of `docs/zh_cn/get_started.md` (#16)
- Add Chinese translation of `docs/zh_cn/tutorials/customize_runtime.md` (#22)
- Add Chinese translation of ` docs/zh_cn/tutorials/customize_config.md` (#23)
- Add Chinese translation of `docs/zh_cn/tutorials/customize_models.md` (#27)
- Add Chinese translation of `docs/zh_cn/model_zoo.md` (#28)
- Add Chinese translation of `docs/zh_cn/faq.md` (#33)

#### Contributors

A total of 13 developers contributed to this release.
Thanks @zytx121 @yangxue0827 @jbwang1997 @liuyanyi @DangChuong-DC @RangeKing @liufeinuaa @np-csu @akmalulkhairin @SheffieldCao @BrotherHappy @Abyssaledge  @q3394101

### CODE REVIEW

**Review by yangxue0827 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -62,7 +62,15 @@ https://user-images.githubusercontent.com/10410257/154433305-416d129b-60c8-44c7-
 
 </details>
 
+## Changelog
+**0.1.1** was released in 18/3/2022:
```
Comment (ZwwWayne) [inline README.md:66]: 14/3

File: docs/en/changelog.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+
+- Support [huge image inference](deom/huge_image_demo.py) (#34)
+- Support HRSC Dataset (#96)
+- Support mixed precision training (#72)
+- Add [colab tutorial](demo/MMRotate_Tutorial.ipynb) for beginners (#66)
+- Add inference speed statistics [tool](tools/analysis_tools/benchmark.py) (#86)
+- Add confusion matrix analysis [tool](tools/analysis_tools/confusion_matrix.py) (#93)
+
+#### Bug Fixes
+- Fix URL error of Swin pretrained model (#111)
```
Comment (ZwwWayne) [inline docs/en/changelog.md:15]: empty line above.

File: README_zh-CN.md [hunk 3]
```diff
@@ -59,6 +59,16 @@ https://user-images.githubusercontent.com/10410257/154433305-416d129b-60c8-44c7-
 
 </details>
 
+## 更新日志
+最新的 **0.1.1** 版本已经在 2022.03.14 发布:
```
Comment (ZwwWayne) [inline README_zh-CN.md:63]: empty line above.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**